server/fixtures/articleValidationData.js

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
There should be no separate file for validations, all these should go into the article file